### DISCUSSION TITLE
Q: 使用Deta方式部署时，评论时间显示有误 || Q: When deploying in Deta mode, the comment time is displayed incorrectly

### DISCUSSION THREAD

**Discussion by LajiPZ** [walinejs/waline#2167] [Q&A] [answered]
### Describe the bug | 问题描述

如题，发送时间只会显示8小时之前
查看后台，记录的评论时间正常
另，Deta上的服务端，似乎最近都没有更新？:(
几种部署方法都看或试过了，我这里只有Deta能做到免费部署，浏览快速，不是很想换（

### Website URL | 问题网站

https://waline-1-k4078585.deta.app/

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

TiDB

---
### Describe the bug | Problem description

As the title states, the sending time will only be displayed 8 hours ago.
Check the background and the recorded comment time is normal.
Also, it seems that the server on Deta has not been updated recently? :(
I have read or tried several deployment methods. Only Deta here can deploy for free, and the browsing is fast. I don’t want to change (

### Website URL | Problem website

https://waline-1-k4078585.deta.app/

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is the data stored?

TiDB

**Comment by lizheming** [comment, marked as answer]
这个是 TiDB 数据库的时区问题，可以去数据库里修改下时区，参考文档 https://docs.pingcap.com/tidb/stable/configure-time-zone

```
SET GLOBAL time_zone = '+8:00';
```
![image](https://github.com/walinejs/waline/assets/424491/533a80df-58a8-4b6c-b82a-0a7d7216544c)


---
This is a time zone issue with the TiDB database. You can modify the time zone in the database. Refer to the document https://docs.pingcap.com/tidb/stable/configure-time-zone

```
SET GLOBAL time_zone = '+8:00';
```
![image](https://github.com/walinejs/waline/assets/424491/533a80df-58a8-4b6c-b82a-0a7d7216544c)
